exporter/jaeger/jaegergrpcexporter/config.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package jaegergrpcexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/config/configmodels"
+)
+
+// Config defines configuration for Jaeger gRPC exporter.
+type Config struct {
```

Comment:
Should we leave a TODO for TLS option in the config?

Reply:
Yes, I will create an issues to track not TLS but also other missing configurations: timeout, headers, and so on.

Reply:
Actually there is already one issue to TLS to jaeger receiver and exporter: https://github.com/open-telemetry/opentelemetry-service/issues/126